assets/src/edit-story/components/library/textLibrary.js

```diff
@@ -14,58 +14,105 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+
 /**
- * External dependencies
+ * Internal dependencies
  */
-import PropTypes from 'prop-types';
+import { DEFAULT_EDITOR_PAGE_HEIGHT } from '../../constants';
+import { editorToDataY } from '../../units/dimensions';
+import { Section, MainButton, Title, SearchInput, Header } from './common';
+import { FontPreview } from './text';
+
+const PRESETS = [
+  {
+    id: 'heading',
+    title: __('Heading', 'web-stories'),
+    fontSize: 48,
+    fontWeight: 800,
+    fontFamily: 'Ubuntu',
+  },
+  {
+    id: 'subheading',
+    title: __('Subheading', 'web-stories'),
+    fontSize: 32,
+    fontWeight: 500,
+    fontFamily: 'Ubuntu',
+  },
+  {
+    id: 'body-text',
+    title: __('Body text', 'web-stories'),
+    fontSize: 16,
+    fontWeight: 'normal',
+    fontFamily: 'Ubuntu',
+  },
+];
+
+const DEFAULT_TEXT_TRANSFORM = {
+  width: 50,
+  height: 20,
+  x: 5,
+  y: 5,
+  rotationAngle: 0,
+};
 
-function MediaLibrary({ onInsert }) {
+function TextLibrary({ onInsert }) {
   return (
     <>
-      <button
-        onClick={() =>
-          onInsert('text', {
-            content: 'Hello',
-            color: 'black',
-            width: 50,
-            height: 20,
-            x: 5,
-            y: 5,
-            rotationAngle: 0,
-          })
-        }
-      >
-        {__('Insert default', 'web-stories')}
-      </button>
-      <br />
-      <button
-        onClick={() =>
-          onInsert('text', {
-            content: 'Hello <strong>World</strong>',
-            color: 'purple',
-            fontSize: 100,
-            fontFamily: 'Ubuntu',
-            fontWeight: 400,
-            width: 50,
-            height: 20,
-            x: 5,
-            y: 5,
-            rotationAngle: 0,
-          })
-        }
-      >
-        {__('Insert big purple ubuntu', 'web-stories')}
-      </button>
+      <Header>
+        <Title>{__('Text', 'web-stories')}</Title>
+        <MainButton
```

Comment:
If there is no such a button, everything  shifts up by 1-2px. It is noticeable when quickly going through all the tabs.

You can test it yourself by toggling `display: none;` on the button in dev tools.